proxy/src/bin/local_proxy.rs

```diff
@@ -124,8 +125,8 @@ async fn main() -> anyhow::Result<()> {
 
     Metrics::install(Arc::new(ThreadPoolMetrics::new(0)));
 
-    info!("Version: {GIT_VERSION}");
-    info!("Build_tag: {BUILD_TAG}");
+    debug!("Version: {GIT_VERSION}");
```

Comment:
I actually like having these in the logs, but could be on one like.

Reply:
This is local proxy only ^^

Reply:
More specifically, the current local_proxy logs show:
```
INFO Version: git:unknown
INFO Build_tag: build_tag-env:6504
```

Neither of these are useful 😅

Reply:
Should we desire to fix this, I think we should move it to compute_ctl instead.